backend/src/email/middlewares/email-transactional.middleware.ts

```diff
@@ -173,9 +223,13 @@ export const InitEmailTransactionalMiddleware = (
       emailMessageTransactional.set('acceptedAt', new Date())
       await emailMessageTransactional.save()
 
+      const ccEmails = await getCCEmails(emailMessageTransactional.id)
```

Comment:
Can we just use `res` object to pass along these data instead of querying from DB again?

Reply:
We can, but by calling getCCEmails we will only return the emails that actually got sent the email (e.i. not in the blacklist) but if we use the value from req, we will get all emails (blacklisted+ non blacklisted)

Reply:
We can right? The value is at line 170

Reply:
yea the value in 170 will have all emails in cc and bcc (submitted by user), but the values in ccEmails are only emails that aren't blacklisted

I can edit to return all emails in cc and bcc if that makes more sense, but I thought it would make more sense to return just the emails that arent in the blacklist